x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/action_delete.test.tsx

```diff
@@ -46,4 +67,22 @@ describe('DeleteAction', () => {
 
     expect(getByTestId('mlAnalyticsJobDeleteButton')).toHaveAttribute('disabled');
   });
+
+  test('When delete modal is open, modal lets user delete target index by default.', () => {
```

Comment:
Usually, `test` suite starts with should, e.g. `should allow to delete target index by default`.
If you have a common arrangement for tests you should combine them in group, for example 
```
describe('When delete model is open', () => {
  beforeEach(async () => {
    // your arrangment here to prepare desired condition  
  })
 
 test('should allow to delete target index by default', async () => {
 })
})
```